front/app/modules/commercial/content_builder/admin/components/RenderNode/index.tsx

```diff
@@ -24,10 +24,14 @@ const RenderNode = ({ render }) => {
 
   const nodeNameIsVisible = isActive && id !== ROOT_NODE;
 
-  const getComponentNameMessage = (name: 'Container') => {
+  const getComponentNameMessage = (name) => {
     switch (name) {
       case 'Container':
         return messages.oneColumn;
+      case 'Text':
+        return messages.text;
+      default:
```

Comment:
@IvaKop As all paths must return a value, a default is needed. However, since a "message" needs to be returned, should I create a new message.defaultComponentName or something which holds an empty string?

Reply:
I see you removed the type from name - can you put it back and add `Text` to it?

Reply:
When I had Text specified there I ended up with an error for the Container case (Type '"Container"' is not comparable to type '"Text"').

Reply:
Yes, because you need to add text to the type `'Container' | 'Text'`

Reply:
Ah I see! Ok, thank you.